src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -264,7 +264,7 @@ public static function getProfile($uid)
 		$data['name'] = $contact['name'];
 		$data['vcard:hasAddress'] = ['@type' => 'vcard:Home', 'vcard:country-name' => $profile['country-name'],
 			'vcard:region' => $profile['region'], 'vcard:locality' => $profile['locality']];
-		$data['summary'] = $contact['about'];
+		$data['summary'] = BBCode::convert($contact['about'], false, 9);
```

Comment:
I would suggest `BBCode::convert($contact['about'], false);`.

Reply:
`$simple_html = 9` is marked for ActivityPub, is it wrong? Inaccurate? What's the use case?

Reply:
The biggest difference is that "9" removes pictures. They are added as attachment later on. But since we here don't transmit attachments, we should transmit the pictures in a regular way. And since this will most likely be used for Friendica<->Friendica as well, I would like to make it as lossless as possible.

Reply:
Fair enough.

Reply:
Are you planning to make this small change?